sklearn/compose/tests/test_column_transformer.py

```diff
@@ -908,6 +911,77 @@ def test_column_transformer_remainder():
     assert ct.remainder == "drop"
 
 
+@pytest.mark.parametrize("force_int", [False, True])
```

Comment:
We need to make sure to not have new warning in the example as well.

Reply:
We should also have a `# TODO` on the top to announce when we are going to remove or change some part of the code.